### ISSUE TITLE
Vox can't latejoin

### ISSUE THREAD

**Issue by Mewchild** [HeavensGate/Eternal#240]
Despite Vox not being whitelisted, they're restricted to joining on roundstart. Not sure if there's a particular reason why this may be the case, or if I'm just slow.

**Comment by Mewchild** [comment]
We uh. Got caught up in needing to work on lore around species, namely that vox shouldn't be heads without the first whitelist in HG history, and the need to increase inclusion of vox in every round - as the yakkity stupid heads they always play - got put on hold until we've fleshed out their place on HG. Since only a certain way allows vox to join, we don't have to worry as much about randoms joining as "shit birds."

tl;dr they aren't whitelisted, this is a pseudo-whitelist tempfix while we get a whitelist in place.
